code/sarix_model/configs/sarix_p8_4rt_thetashared_sigmanone_xmas_spike.py

```diff
@@ -0,0 +1,7 @@
+import copy
+from configs.base import base_config
+
+config = copy.deepcopy(base_config)
```

Comment:
If you ever want to dive into Python dataclasses, this might be a good place--you could define `base_config` as a dataclass and then instantiate it with model_name, p, power_transform, etc. as needed